bin/anthology/volumes.py

```diff
@@ -118,7 +161,7 @@ def get_title(self, form="xml"):
           - plain: Strip all XML tags, returning only plain text
           - html:  Convert XML tags into valid HTML tags
         """
-        return self.formatter(self.get("xml_title"), form)
+        return self.formatter(self.get("xml_booktitle"), form)
```

Comment:
Can you explain this change? Maybe it's obvious but I don't quite follow.

Reply:
The title of a volume is `booktitle`, stored in the metadata. I think the volume title used to be read in as the `title` field from paper ID 0.

Reply:
Ah, right, this is coming from the metadata block now. Got it, I think.